Source/WebCore/page/scrolling/ScrollSnapOffsetsInfo.h

```diff
@@ -38,12 +38,15 @@ namespace WebCore {
 class ScrollableArea;
 class RenderBox;
 class RenderStyle;
+class Element;
 
 template <typename T>
 struct SnapOffset {
     T offset;
     ScrollSnapStop stop;
     bool hasSnapAreaLargerThanViewport;
+    unsigned snapTargetId;
```

Comment:
Name nitpick: This is written as "ID" everywhere else, but "Id" here. :slightly_smiling_face:

Reply:
Fixed.